tests/CbtThemeLocale/escapeString.php

```diff
@@ -13,29 +13,29 @@ class CBT_Theme_Locale_EscapeString extends CBT_Theme_Locale_UnitTestCase {
 	public function test_escape_string() {
 		$string         = 'This is a test text.';
 		$escaped_string = CBT_Theme_Locale::escape_string( $string );
-		$this->assertEquals( "<?php echo __('This is a test text.', 'test-locale-theme');?>", $escaped_string );
+		$this->assertEquals( "<?php esc_html_e('This is a test text.', 'test-locale-theme');?>", $escaped_string );
```

Comment:
Unrelated to the changes, but why are we using CamelCase for these test files? It'd be better if we kept the convention we're already using for the other files. Same for the directory name.

Reply:
That's the core convention. Example: https://github.com/WordPress/wordpress-develop/tree/trunk/tests/phpunit/tests/blocks

Reply:
> if we kept the convention we're already using for the other files. Same for the directory name.
Those should be progressively updated when we have the chance.

Reply:
I submitted an issue for standardizing the convention used on tests: https://github.com/WordPress/create-block-theme/issues/666

Reply:
I don't think those tests follow the actual convention either:

https://make.wordpress.org/docs/style-guide/formatting/filenames/#naming-files

> Use lowercase file, folder, and directory names. In general, separate words in filenames with hyphens, not underscores. Use standard [ASCII alphanumeric characters](https://wikipedia.org/wiki/ASCII#Character_set) in file, folder, and directory names.

Reply:
> Use lowercase file, folder, and directory names.

I'm not sure because, as far as I saw, the are numerous core test files using camel case.

Example 2: https://github.com/WordPress/wordpress-develop/tree/trunk/tests/phpunit/tests/blocks
Example 1: https://github.com/WordPress/wordpress-develop/tree/trunk/tests/phpunit/tests/fonts/font-face